src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -1052,14 +1057,17 @@ protected void UpdatePanelSizing(float delta)
     protected abstract void UpdateAbilitiesHotBar();
 
     protected void UpdateBaseAbilitiesBar(bool showEngulf, bool showToxin, bool showSlime,
-        bool showingSignaling, bool engulfOn, bool showEject)
+        bool showingSignaling, bool engulfOn, bool sprintOn, bool showEject)
     {
+        sprintHotkey.Visible = stage?.GameWorld.WorldSettings.ExperimentalFeatures == true;
```

Comment:
Instead of allowing stage to be null here, I think a better thing to do would be to just flat out:
```
if (stage == null)
    throw new InvalidOperationException("Stage must be set before abilities update");
```